basics/README.md

```diff
@@ -0,0 +1,16 @@
+# Goals
+* Understand Basic Kubernetes (k8s) Concepts
+* Familiarize with the different parts of a k8s manifest File
+* Deploy an application in ~~local~~ k8s
+* ~~Interact with Kubernetes locally~~
```

Comment:
I'm curious as to why these are struck out?  Is this a way to indicate that we aren't doing something?